ShopifySharp/Services/FulfillmentService/FulfillmentServiceService.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Newtonsoft.Json.Linq;
+using System.Net.Http;
+using ShopifySharp.Filters;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ShopifySharp.Infrastructure;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// A service for manipulating Shopify fulfillments.
+    /// </summary>
+    public class FulfillmentServiceService : ShopifyService
+    {
+        /// <summary>
+        /// Creates a new instance of <see cref="FulfillmentServiceService" />.
+        /// </summary>
+        /// <param name="myShopifyUrl">The shop's *.myshopify.com URL.</param>
+        /// <param name="shopAccessToken">An API access token for the shop.</param>
+        public FulfillmentServiceService(string myShopifyUrl, string shopAccessToken) : base(myShopifyUrl, shopAccessToken) { }
+
+        /// <summary>
+        /// Gets a list of your app's FulfillmentServices.
+        /// </summary>
+        /// <param name="options">Options for filtering the list. Set scope to all to retrieve all of the store's fulfillment services</param>
+        /// <returns>The list of fulfillment services matching the filter.</returns>
+        public virtual async Task<IEnumerable<FulfillmentServiceEntity>> ListAsync(ListFilter filter = null)
```

Comment:
Change `ListFilter filter = null` to `FulfillmentServicesFilter filter = null` once that class has been created.